app/views/content/terms_candidate.html.erb

```diff
@@ -83,7 +83,7 @@
 
     <p class="govuk-body">You’ll be asked about convictions in the application form.</p>
```

Comment:
Is this a duplicate, given that we have a line directly above it which says "You’ll be asked about any criminal convictions in the application form."

Reply:
yeh, remove